### ISSUE TITLE
Galacticraft Oxygen Bubble Distributor sometimes stops delivering oxygen to a player when they rejoin

### ISSUE THREAD

**Issue by Ramog** [xJon/Tekkit-2#267]
### Was this bug reported before?

- [X] No, it was not.

### Please describe the issue you are experiencing

In version 1.2.3 and 1.2.4  of Tekkit 2 I could reproduce a bug with the Oxygen Bubble Distributor from Galacticraft.
When you leave the server in or near the bubble and rejoin it does not recognize that you should have oxygen and you slowly sufficate (same as if it would not be there).

Same goes for teleporting out of the dimension and telporting back.

**Comment by xJon** [comment]
Hi! Can you reproduce this issue in a test singleplayer world (it's not server-dependent?)
Also, can you confirm whether this issue was present in versions before `1.2.3`? This could be helpful in pinpointing the cause!

**Comment by Ramog** [comment]
> Hi! Can you reproduce this issue in a test singleplayer world?

Yeh I was able to, althought I needed mystcraft to use its  dimensional teleportation command.
It definitly has a weird condition to triggering it but leaving the server and joining does cause it easily on a normal server and thats completly tekkit 2 native.
Someone should also do some research if that happens with normal chunk-loading/-unloading too but I already put as much work into it as I am willing and able to and chunkloading stuff is very weird to debug since there are allot of edge cases.

> Also, was this issue not present in versions before `1.2.3`?

I don't know 1.2.3 and 1.2.4 was all I was able to test since I did it for server and client (that the /tpx command of mystcraft triggered it told me that there are atleast cases that can trigger it even on singleplayer).

But the things I tested where able to produce the issue consistantly.

**Comment by xJon** [comment]
Testing it on a server, are you able to reproduce this consistently?
I just tried a couple of times, and managed to reproduce it only once but not the other time.

By the way, you can use `/forge setdim username dimid` instead of Mystcraft's `/tpx`!

Edit: I tested again (including teleporting with `/forge setdim`) and I can't seem to get it to happen. Weird.

**Comment by Ramog** [comment]
> Testing it on a server, are you able to reproduce this consistently? I just tried a couple of times, and managed to reproduce it only once but not the other time.
> 
> By the way, you can use `/forge setdim username dimid` instead of Mystcraft's `/tpx`!
> 
> Edit: I tested again (including teleporting with `/forge setdim`) and I can't seem to get it to happen. Weird.

for me it happend quite often on a server, often enough where I would say it is a problem but eh, I just wanted to report it, I don't have things setup so I could easily test it anymore now. ^^'

**Comment by xJon** [comment]
~~No worries! Any chance you could also report it over https://github.com/TeamGalacticraft/Galacticraft-Legacy/issues?~~

https://github.com/TeamGalacticraft/Galacticraft-Legacy/issues/217 👍

**Comment by xJon** [comment]
As Galacticraft is not in active development, closing for now as `wontfix`.
